app/controllers/benchmark_runs_controller.rb

```diff
@@ -27,10 +27,22 @@ def create
       benchmark_result_type: benchmark_result_type
     )
 
+    if params[:commit_hash]
+      commit_objects = BenchmarkRun.where( initiator: initiator, benchmark_type: benchmark_type,
+                       benchmark_result_type: benchmark_result_type).limit(10)
```

Comment:
The number 10 should not be hard coded. I know it's still WIP but just noting it down :smile: